packages/core/src/utils/session.ts

```diff
@@ -0,0 +1,23 @@
+import {storage, Storage} from '../storage/Storage';
+
+class SessionStorage {
+  private key: string;
+
+  constructor(private localStorage: Storage) {
+    this.key = 'test';
+    this.recrypt();
+  }
+
+  insert({data}: {data: any}) {
+    this.localStorage.setItem(this.key, data);
+  }
+  getItem() {
+    const item = this.localStorage.getItem(this.key);
+    return item;
+  }
```

Comment:
As this is an interface to get sessions, I think we could put function names with more meaning
addSession
getSession

Also, please add type for a session object to define each needed props, I think I added it in the ticket..